onload/build/diagnostics/Dockerfile

```diff
@@ -2,9 +2,12 @@
 # SPDX-FileCopyrightText: (c) Copyright 2023 Advanced Micro Devices, Inc.
 FROM image-registry.openshift-image-registry.svc:5000/onload-clusterlocal/onload-user:v8.1.0 AS onload-user
 
-FROM ubi8:8.8
+FROM openshift4/ose-cli
```

Comment:
To comply with our 'all images must be tagged' policy, the tag in this case may need to be an env var, or we just fix `v4.12.0` for now (as `kubectl` supports n-2, so should support v4.10 too).

Reply:
Have chosen `v4.12.0` for now.